examples/disk_offline_inference/example.yaml

```diff
@@ -0,0 +1,7 @@
+chunk_size: 256
+local_device: "file:///local/qinyuyang2003/local_disk/"
```

Comment:
Can we change this to "file://local_folder", which means writing to the folder named `local_folder` under the current directory? (Usually, there won't be a folder named `/local/qingyuyang2003/local_disk` on user's machine)

And add the instruction of `mkdir -p local_folder` in the README.md to let users create a folder at the beginning.

Reply:
we will create the folder for them